### ISSUE TITLE
bug: Ion-date-time:  Current a11y issues to test when rewritten for desktop

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic-framework#22711]
This bug is just a note as we work towards the desktop milestone that the current ion-date-time has an a11y issue whereby AXE reports that all buttons should have discernible text.

We need to review that they either do or that there is a reason that they do not.

Given that we are looking at a substantial re-write with this component this is less of a bug perhaps but more of a note of things to review at the proper time.

Also, ensure that keyboard action works with new date time. Does not currently work.

**Comment by liamdebeasi** [comment]
Confirmed that this is resolved with the new `ion-datetime` component. Thanks!
